docker/cluster-compose.yml

```
@@ -0,0 +1,146 @@
+services:
+  node0:
+    image: muhammadqadora/falkor-redis
+    container_name: node0
+    hostname: node0
+    ports:
+      - 5000:5000
+    networks:
+      - redis-cluster
+    healthcheck:
+          test: |
```

yamllint (1.35.1)

[warning] wrong indentation: expected 6 but found 10

(indentation)